splinterd/src/config/partial.rs

```diff
@@ -671,19 +671,19 @@ impl PartialConfig {
     }
 
     #[cfg(feature = "oauth")]
-    /// Adds an `with_oauth_openid_url` value to the `PartialConfig` object.
+    /// Adds an `oauth_openid_url` value to the `PartialConfig` object.
```

Comment:
This commit could go into a separate PR as it is very straight-forward and unrelated to the larger config changes.